qcodes/instrument_drivers/Gentec/Gentec_Maestro.py

```diff
@@ -3,7 +3,7 @@
 
 
 class Gentec_Maestro(VisaInstrument):
-    """
+    r"""
```

Comment:
what is `r` needed for?

Reply:
In its most general form it is related to this issue:
 https://github.com/sphinx-doc/sphinx/issues/4479

In this case, I successfully traced back the error to the point where I removed an "r" in visa.py as I thought it was unnecessary. I set it back (see commit 7c24ccf) and that fixed it while I let this one also stay as I did not want to make another unnecessary commit and it does not do any harm.

Reply:
So to  add a bit more technical detail. The r marks the string as raw this means that rather interpreting `\n` as a new line it will be the literal string `\n` This is needed for the docstring of visa since it contains the terminator docs. I don't think it would be needed here but indeed as long as you don't want to write any none escaped sequences it does no harm